src/components/font_size_editor/font_size_editor.ts

```diff
@@ -1,19 +1,20 @@
 import { Component, useExternalListener, useRef, useState } from "@odoo/owl";
-import { setStyle } from "../../actions/menu_items_actions";
 import { DEFAULT_FONT_SIZE, FONT_SIZES, SELECTION_BORDER_COLOR } from "../../constants";
 import { clip } from "../../helpers/index";
 import { SpreadsheetChildEnv } from "../../types/index";
 import { css } from "../helpers/css";
 import { isChildEvent } from "../helpers/dom_helpers";
+import { Popover, PopoverProps } from "../popover";
 
 interface State {
   isOpen: boolean;
 }
 
 interface Props {
-  onToggle: () => void;
-  dropdownStyle: string;
+  currentFontSize?: number;
```

Comment:
Initially, I hadn't marked this as optional. However, I made it optional later because, during tests, mounting the title component required passing fontSize every time, which would otherwise cause the tests to break.